src/webui/www/public/scripts/client.js

```diff
@@ -310,13 +312,19 @@ window.addEvent('load', function () {
                         update_categories = true;
                     }
                     if (response['torrents']) {
+                        var newTorrentInList = false;
```

Comment:
my point is, if there is a concept of new to the list, there must be something old/existing already in the list, but there isn't one in this block scope.